src/main/java/org/r0bb3n/maven/SonarQualityGateMojo.java

```diff
@@ -164,9 +171,13 @@ public void execute() throws MojoExecutionException, MojoFailureException {
       String failedConditions = projectStatus.getConditions().stream()
           .filter(has(ProjectStatus.Status.OK, ProjectStatus.Status.NONE).negate())
           .map(c -> c.getMetricKey() + ":" + c.getStatus()).collect(Collectors.joining(", "));
-      throw new MojoFailureException(String
-          .format("Quality Gate not passed (status: %s)! Failed metric(s): %s",
-              projectStatus.getStatus(), failedConditions));
+      String message = String.format("Quality Gate not passed (status: %s)! Failed metric(s): %s",
+          projectStatus.getStatus(), failedConditions);
+      if (!testFailureIgnore) {
+        throw new MojoFailureException(message);
+      } else {
+        getLog().info(message);
```

Comment:
Please change to `warn`, since this is more than `info`
```suggestion
        getLog().warn(message);
```